packages/node/test/fixtures/evm/event-logs.ts

```diff
@@ -13,11 +13,11 @@ export function buildClientRequest(overrides?: Partial<Log>): Log {
     removed: false,
     address: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
     data:
-      '0x00000000000000000000000000000000000000000000000000000000000000020000000000000000000000009fe46736679d2d9a65f0992f2272de9f3c7fa6e0e315dcd8305800ebdf4c188fa85c602387d36df23de6927d28820d695a3c0deb0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a46c4b41d72ada9d14157b28a8a2db97560fff120000000000000000000000009fe46736679d2d9a65f0992f2272de9f3c7fa6e048a4157c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000060316200000000000000000000000000000000000000000000000000000000000066726f6d000000000000000000000000000000000000000000000000000000004554480000000000000000000000000000000000000000000000000000000000',
+      '0x00000000000000000000000000000000000000000000000000000000000000020000000000000000000000009fe46736679d2d9a65f0992f2272de9f3c7fa6e0537522dde3d2d4fb409b9c278d0b08de85f5aea7339efd23c5a8d7316242d2db0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000a46c4b41d72ada9d14157b28a8a2db97560fff120000000000000000000000009fe46736679d2d9a65f0992f2272de9f3c7fa6e048a4157c0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000060316200000000000000000000000000000000000000000000000000000000000066726f6d000000000000000000000000000000000000000000000000000000004554480000000000000000000000000000000000000000000000000000000000',
```

Comment:
I just found and replaced the `templateId` here. To what you want with this information.